### DISCUSSION TITLE
Forecast issues

### DISCUSSION THREAD

**Discussion by brewin** [weather-gov/api#724] [General]
https://api.weather.gov/gridpoints/BOI/112,80/forecast
```Json
      "periods": [
            {
                "number": 1,
                "name": "Today",
                "startTime": "2024-06-04T11:00:00-06:00",
                "endTime": "2024-06-04T18:00:00-06:00",
                "isDaytime": true,
                "temperature": 83,
                "temperatureUnit": "F",
                "temperatureTrend": null,
                "probabilityOfPrecipitation": {
                    "unitCode": "wmoUnit:percent",
                    "value": null
                },
                "dewpoint": {
                    "unitCode": "wmoUnit:degC",
                    "value": 10.555555555555555
                },
                "relativeHumidity": {
                    "unitCode": "wmoUnit:percent",
                    "value": 47
                },
                "windSpeed": "7 to 10 mph",
                "windDirection": "W",
                "icon": "https://api.weather.gov/icons/land/day/sct?size=medium",
                "shortForecast": "Mostly Sunny",
                "detailedForecast": "Mostly sunny, with a high near 83. West wind 7 to 10 mph."
            },
```

Temperature is in F and has a separate `temperatureUnit` instead of `unitCode`. Dew point is in C. Wind speed is in MPH and is a string instead of being separate speed and gust values with `unitCode` properties. Wind direction should probably be in degrees. Floating point numbers should be rounded. Barometric pressure is missing. 

The hourly forecast has similar issues. All these issues make the API very difficult to use.

**Comment by dwhitemv25** [comment]
Try setting the header
```
Feature-Flags: forecast_temperature_qv,forecast_wind_speed_qv
```
That will change the temperature and wind speed elements to use the same QuantiativeValue schema that the other forecast elements use.

**Reply by brewin** [reply]
Are these feature flags documented anywhere? The [update page](https://www.weather.gov/documentation/services-web-api) just says "There are currently no feature flags."

**Reply by dwhitemv25** [reply]
They are documented on the Specification tab under the [/gridpoints/{wfo}/{x},{y}/forecast](https://www.weather.gov/documentation/services-web-api#/default/gridpoint_forecast) endpoint. I'm guessing the Updates tab feature flags are for global flags, while these are endpoint-specific.

**Comment by brewin** [comment]
To anyone able to edit the documentation, "Feature-Flag" on the Overview tab should be "Feature-Flags". The singular does not work.
